tests/load/locustfile.py

```diff
@@ -132,7 +227,12 @@ def test_basic_topic(self):
         assert (
             endpoint_res.status_code == 201
         ), f"status code was {endpoint_res.status_code}"
-        # connect and check notification
+
+        # NOTE: To properly test "topic" messages, we really ought to
```

Comment:
This almost sounds like a TODO but `test_connect_stored` does this. Maybe reword or maybe mention it here?

Reply:
`test_connect_stored` kind of does that.

What it does is send in 10 topic messages, then read from the queue 10 times looking to see if the response matches. Since the response is never ACK'd, it's never removed, and all 10 work correctly, but it feels like that's a bug rather than a feature.

Reply:
Ah, my mistake